scripts/generate_key.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+set -e
+
+KEY_FORMAT=$1 # APT, RPM or TARGZ
+
+if [ -z $KEY_FORMAT ];then
+  echo -e "${RED} usage ./generate_key.sh [APT|RPM|TAGGZ] .${NC}"
```

Comment:
I'm not sure `${RED}` and the likes are portable across shells.

Reply:
Ah, I see now they are defined below. Will that work? I would expect them to be defined before they are used.